scripts/ci/jobs/jobs.yml

```diff
@@ -373,6 +373,24 @@
           results: tests.xml
       - chuck-norris
 
+- job:
+    name: skydive-export-sa-tests
+    defaults: skydive
+    triggers:
+      - skydive-pull-request:
+          name: functional-tests-export-sa
+          trigger-prefix: "all "
```

Comment:
Should be :
`trigger-prefix: "(all )?"`

Reply:
done